### ISSUE TITLE
SPF Plugin: AddressValueError if 'ipaddress' instead of 'ipaddr' is installed

### ISSUE THREAD

**Issue by gryphius** [gryphius/fuglu#46]
```
2015-12-24 14:47:18,337 fuglu.SessionHandler: ERROR Plugin SPF Check failed: Traceback (most recent call last):
  File "/usr/lib/python2.6/site-packages/fuglu/scansession.py", line 224, in run_plugins
    ans = plugin.examine(suspect)
  File "/usr/lib/python2.6/site-packages/fuglu/plugins/domainauth.py", line 321, in examine
    result, explanation = spf.check2(ip, suspect.from_address, helo)
  File "/usr/lib/python2.6/site-packages/pyspf-2.0.12-py2.6.egg/spf.py", line 315, in check2
    receiver=receiver,timeout=timeout,verbose=verbose,querytime=querytime).check()
  File "/usr/lib/python2.6/site-packages/pyspf-2.0.12-py2.6.egg/spf.py", line 397, in __init__
    self.set_ip(i)
  File "/usr/lib/python2.6/site-packages/pyspf-2.0.12-py2.6.egg/spf.py", line 424, in set_ip
    self.ipaddr = ipaddress.ip_address(i)
  File "/usr/lib/python2.6/site-packages/ipaddress-1.0.15-py2.6.egg/ipaddress.py", line 163, in ip_address
    ' a unicode object?' % address)
AddressValueError: '74.125.82.54' does not appear to be an IPv4 or IPv6 address. Did you pass in a bytes (str in Python 2) instead of a unicode object?
```

We shold probably try to handle both libraries

**Comment by gryphius** [comment]
this is actually a bug in pyspf itself which should be fixed with pyspf 2.0.12  ( https://sourceforge.net/p/pymilter/bugs/41/ )

**Comment by danBLA** [comment]
I know this is closed but just because I ran again into it:
On Centos-7 (using epel repository), pyspf 2.0.11 is still default. To install
pyspf 2.0.12 run which will get the latest version of 2.0.12:

`pip2 install pyspf==2.0.12t`
